src/azure-cli-core/azure/cli/core/tests/recordings/test_completion.yaml

```diff
@@ -0,0 +1,47 @@
ts","sku":{"name":"Standard_GRS","tier":"Standard"},"kind":"StorageV2","location":"westus","tags":{}},{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/azpslrg3hpv1tozfa/providers/Microsoft.Storage/storageAccounts/azpsl5za0k8m9eq","name":"azpsl5za0k8m9eq","type":"Microsoft.Storage/storageAccounts","sku":{"name":"Standard_GRS","tier":"Standard"},"kind":"StorageV2","location":"westus","tags":{}},{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/azpslrg90o3nqqr33/providers/Microsoft.Storage/storageAccounts/azpsl3ucg42p3zn","name":"azpsl3ucg42p3zn","type":"Microsoft.Storage/storageAccounts","sku":{"name":"Standard_GRS","tier":"Standard"},"kind":"StorageV2","location":"westus","tags":{}},{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitestm5i4qofcc6eb7w4lgmppsi2r7riovbkygfshnuxpz3qfzr26mgv5apdrerakyf2jqtk5/providers/Microsoft.Storage/storageAccounts/versionqyz6scpkfghqeoti4","name":"versionqyz6scpkfghqeoti4","type":"Microsoft.Storage/storageAccounts","sku":{"name":"Standard_LRS","tier":"Standard"},"kind":"StorageV2","location":"centraluseuap","tags":{}},{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rgvszkkwmpogaa6jjkwjdoyqwwl2ckzd43nogod7o6moiweutkohk6inxg3supcuubx/providers/Microsoft.Storage/storageAccounts/clitest4colel7tolw26rjxy","name":"clitest4colel7tolw26rjxy","type":"Microsoft.Storage/storageAccounts","sku":{"name":"Standard_LRS","tier":"Standard"},"kind":"StorageV2","location":"eastus2euap","tags":{}},{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest53hawxoqrfrf2qu5dyoqddepgzs7fzjzc3s7oamsllnwgh5rc6sbrxd54j5agmhfyhut/providers/Microsoft.Storage/storageAccounts/clitest77tgjxr7mgozmqiik","name":"clitest77tgjxr7mgozmqiik","type":"Microsoft.Storage/storageAccounts","sku":{"name":"Standard_LRS","tier":"Standard"},"kind":"StorageV2","location":"centraluseuap","tags":{}},{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/acctestRG-networkfw-xuwu1/providers/Microsoft.Storage/storageAccounts/acctestaccx4vvg","name":"acctestaccx4vvg","type":"Microsoft.Storage/storageAccounts","sku":{"name":"Standard_LRS","tier":"Standard"},"kind":"StorageV2","location":"westus","identity":{"type":"None"},"tags":{}},{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/azpssmokerg56fa2ss4td/providers/Microsoft.Storage/storageAccounts/azpssmokesa56fa2ss4td","name":"azpssmokesa56fa2ss4td","type":"Microsoft.Storage/storageAccounts","sku":{"name":"Standard_LRS","tier":"Standard"},"kind":"StorageV2","location":"westus","tags":{}},{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/azpslrg05jvw42i2l/providers/Microsoft.Storage/storageAccounts/azpslsa7gub1mfoyq","name":"azpslsa7gub1mfoyq","type":"Microsoft.Storage/storageAccounts","sku":{"name":"Standard_GRS","tier":"Standard"},"kind":"StorageV2","location":"westus","tags":{"Az.Sec.DisableAllowSharedKeyAccess::Skip":"For
```

Comment:
This recording contains many internal testing resource names. Can we avoid this?